api/artillery-load-test.yml

```diff
@@ -0,0 +1,62 @@
+config:
+  target: http://localhost:8080
+  tls:
+    rejectUnauthorized: false
+  http:
+    timeout: 10
+  phases:
+    - duration: 30
+      arrivalRate: 2
+      maxVusers: 50
+      name: Warm up phase
+    - duration: 30
+      arrivalRate: 28
+      maxVusers: 250
+      name: Ramp up load
+    - duration: 30
+      arrivalRate: 56
+      maxVusers: 1000
+      name: Spike phase
```

Comment:
I would prefer that happen as a part of https://github.com/HHS/simpler-grants-gov/issues/1390. We should use this configuration to scale to the actual amount we expect to handle.

Mostly I'm being bullish about closing this ticket because its one of the last tickets in the Get Opp V0 30k

Reply:
I added an explicit note to the acceptance criteria for https://github.com/HHS/simpler-grants-gov/issues/1390 that we should adjust this configuration